src/app/ui/editor/editor.cpp

```diff
@@ -2363,6 +2363,10 @@ void Editor::onTiledModeChange()
 
   spritePos += mainTilePosition();
   screenPos = editorToScreen(spritePos);
+  
```

Comment:
This line includes trailing whitespaces, check the pre-commit check:
https://github.com/aseprite/aseprite/actions/runs/13473873443/job/37752188196#step:4:117

Remember to configure the pre-commit hook: https://github.com/aseprite/aseprite/blob/main/CONTRIBUTING.md#pre-commit-hooks